radixdlt/src/main/java/com/radixdlt/consensus/validators/ValidatorSet.java

```diff
@@ -36,11 +36,23 @@
  * as long as all validators sign.
  */
 public final class ValidatorSet {
+	// We assume that we won't have more than 2^128 validators (2^32 is in fact the limit)
+	// in a single validator set so having this as the max power value will prevent overflows
+	private static final UInt256 POWER_MAX_VALUE = UInt256.from(UInt128.MAX_VALUE);
```

Comment:
See other comments on validator power.  Should either make this a UInt128, or a UInt256, but either way allow the full range of the underlying type.

Reply:
yup resolved in https://github.com/radixdlt/radixdlt-core/pull/82